src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -293,6 +288,11 @@ void Tailsitter::fill_actuator_outputs()
 	if (_vtol_schedule.flight_mode == vtol_mode::FW_MODE) {
 		mc_out[actuator_controls_s::INDEX_THROTTLE] = fw_in[actuator_controls_s::INDEX_THROTTLE];
 
+		/* allow differential thrust if enabled */
+		if (_params->diff_thrust == 1) {
```

Comment:
I suggest moving this change to the top to keep it clean (only responsible for determining the throttle)

```
if (_vtol_schedule.flight_mode == vtol_mode::FW_MODE && _params->diff_thrust == 1)) {
mc_out[actuator_controls_s::INDEX_ROLL] = fw_in[actuator_controls_s::INDEX_YAW] * _params->diff_thrust_scale;
} else {
mc_out[actuator_controls_s::INDEX_ROLL]  = mc_in[actuator_controls_s::INDEX_ROLL]  * _mc_roll_weight;
}
```

Reply:
For tailsitter users, when should I enable this parameter and when should I disable it?

Reply:
I don't have a strong opinion for one or the other way, both work.

Reply:
>For tailsitter users, when should I enable this parameter and when should I disable it?

You should enable it when your vehicle has bad passive yaw stability or you want to fly aggressive maneuvers.